django/views/generic/base.py

```diff
@@ -77,6 +78,12 @@ def view(request, *args, **kwargs):
         # and possible attributes set by decorators
         # like csrf_exempt from dispatch
         update_wrapper(view, cls.dispatch, assigned=())
+
+        # Remove "self" from function signature
+        view_signature = signature(view)
+        view_signature = view_signature.replace(parameters=tuple(view_signature.parameters.values())[1:])
+        view.__signature__ = view_signature
```

Comment:
If we use `import inspect`, we can make this a bit simpler:
```suggestion
        # Remove "self" from the generated view function signature.
        signature = inspect.signature(view)
        parameters = tuple(signature.parameters.values())[1:]
        view.__signature__ = signature.replace(parameters=parameters)
```

Reply:
Yes. import inspect makes it more simple!